pkgs/tools/security/oath-toolkit/default.nix

```diff
@@ -19,8 +19,6 @@ in stdenv.mkDerivation rec {
 
   configureFlags = lib.optionals stdenv.isDarwin [ "--disable-pam" ];
 
-  passthru.updateScript = ./update.sh;
```

Comment:
To let `nixpkgs-update` update the package, do this:
```nix
passthru.updateScript = nix-update-script { attrPath = pname; };
```

Which, as the name gives away, uses `nix-update` :)